process-engine/bpmn-studio
# ✨ Add Unsupported Flag to Elements

## Changes

1. Add Unsupported BPMN-Element List
2. Add Supported BPMN Elements List

PR: #1903

## How to test the changes

- Use [this](https://github.com/process-engine/bpmn-studio/files/3960325/Elements.bpmn.zip) diagram
- Click on all unsupported elements (lower lane)
- **Notice that '[Unsupported]' will appear next to the element type in the Property Panel.**
- Click on all supported elements
- **Notice that no '[Unsupported]' will appear next to the element type in the Property Panel.**
